public/app/features/canvas/runtime/scene.tsx

```diff
@@ -226,7 +226,7 @@ export class Scene {
         if (this.div) {
           this.initMoveable(true, this.isEditingEnabled);
         }
-      }, 100);
+      });
```

Comment:
Removing `100` from `setTimeout` related to reseting `initMoveable` helps with the flashing UX on re-selection (makes it more seamless / does not negatively impact functionality)